.github/workflows/elixir_tests.yml

```diff
@@ -7,7 +7,7 @@ jobs:
     runs-on: ubuntu-20.04
 
     container:
-      image: hexpm/elixir:1.17.0-erlang-27.0-debian-bookworm-20240612
+      image: hexpm/elixir:1.18.0-erlang-27.2-debian-bullseye-20241202
```

Comment:
Can you find a "bookworm" debian release image? bullseye is not the current stable release of debian

Reply:
https://hub.docker.com/layers/hexpm/elixir/1.18.0-erlang-27.2-debian-bookworm-20241202/images/sha256-c108c9e629f6a1aa4f97c0d8f0466d3bae52f1d724cdc460d6278b67979d1f88

Reply:
Thank you :)